modules/python/src/custom_nodes/google_genmedia/veo3_api.py

```diff
@@ -106,17 +106,21 @@ def generate_video_from_text(
         """
         if not prompt or not isinstance(prompt, str) or len(prompt.strip()) == 0:
             raise ValueError("Prompt cannot be empty for text-to-video generation.")
-        if duration_seconds != 8:
+        valid_durations = (4, 6, 8)
+
+        if duration_seconds not in valid_durations:
```

Comment:
Minor nit, use an empty line before the valid values, and no empty line between the values and `if` block.

```suggestion

        valid_durations = (4, 6, 8)
        if duration_seconds not in valid_durations:
```